src/components/TransactionInfo.js

```diff
@@ -35,6 +36,7 @@ const TransactionInfo = ({ info, receipt }) => (
       <dd>{info.input}</dd>
     </dl>
     {receipt && receipt.logs.length > 0 && <TransactionLogs logs={receipt.logs} />}
+    {trace && trace.length > 0 && <TransactionTrace trace={trace} />}
```

Comment:
I think for both of these it'd be nice to say "no trace" or "no logs" when these are 0.